src/Explorer/Notebook/NotebookClientV2.ts

```diff
@@ -34,6 +34,7 @@ import * as TelemetryProcessor from "../../Shared/Telemetry/TelemetryProcessor";
 import { userContext } from "../../UserContext";
 import configureStore from "./NotebookComponent/store";
 import { CdbAppState, makeCdbRecord } from "./NotebookComponent/types";
+import HTML from "./NotebookRenderer/outputs/html";
```

Comment:
>HTML [](start = 7, length = 4)

nit: diff name instead of using gneric name- ma be something liek SanitizedHTML?